src/main/java/seedu/address/model/person/Times.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents how many times a book is borrowed before.
+ * Guarantees: immutable; is always valid
+ */
+public class Times {
+    public final String value;
+
+    /**
+     * Constructs a {@code Times}.
+     *
+     * @param times A valid times.
+     */
+    public Times(String times) {
+        requireNonNull(times);
+        value = times;
```

Comment:
Maybe there can also be an upper bound for the number of times that a book can be borrowed.